.github/workflows/code-coverage.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       name: Set up Go 1.x
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17.12
+        go-version: 1.18.4
```

Comment:
another dumb question... 

what if a new version (or CVE fix) comes out? Will you manually update these files?

```
The version in UBI8 image is: 1.18.4-1.module+el8.7.0+16015+724888d8, for what it's worth. :D 
```

Reply:
Our current approach is "update when Go breaks compatibility". Since these are just CI jobs for checking format and running tests, it's not crucial that they're as current as possible -- image builds still occur in the auto-updated builder images.